SSSCS-8504:Populate venue field with the venue a case is expected to be managed from at creation

rasikasri opened the pull request:
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/SSCS-8504

### Change description ###



**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

jackmaloney reviewed (changes requested):
Please combine the new functional test into the existing functional test. Also, can you add a test to the SscsBulkScanValidateRecordCallback class please?

rasikasri commented:
> Please combine the new functional test into the existing functional test. Also, can you add a test to the SscsBulkScanValidateRecordCallback class please?

Moved the new functional tests into the existing test methos in TransformationFunctionalTest. Also added an assertion into the SscsBulkScanValidateRecordCallback for processingVenue.

jackmaloney commented on src/functionalTest/resources/exception/all_fields_entered_uc.json:
```diff
@@ -87,46 +87,6 @@
       "value": "{PERSON1_NINO}"
     },
-    {
-      "name": "person2_title",
```
Were you supposed to remove these fields from this test?
